Update public cache content (#3710)
magento/devdocs: guides/v2.1/extension-dev-guide/cache/page-caching/public-content.md

Magento disables page caching if at least one non-cacheable block is present in the layout.

{: .bs-callout .bs-callout-warning }

> Caching will be disabled in all the pages of the site when `cacheable="false"` is used inside the `default.xml`.

Reviewer: ```suggestion
Using `cacheable="false"` inside the `default.xml` file disables caching for all pages on the site.
```
